test/Identity.FunctionalTests/ManagementTests.cs

```diff
@@ -52,5 +56,25 @@ public ManagementTests(ServerFactory serverFactory)
             var email = emails.SentEmails[1];
             await UserStories.ConfirmEmailAsync(email, client);
         }
+
+        [Fact]
+        public async Task CanChangePassword()
+        {
+            // Arrange
+            var server = ServerFactory.CreateDefaultServer();
+
+            var client = ServerFactory.CreateDefaultClient(server);
+            var newClient = ServerFactory.CreateDefaultClient(server);
+
+            var userName = $"{Guid.NewGuid()}@example.com";
+            var password = $"!Test.Password1$";
+
+            var index = await UserStories.RegisterNewUserAsync(client, userName, password);
+            var manage = await index.ClickManageLinkAsync();
+            var changePassword = await manage.ClickChangePasswordLinkAsync();
+
+            var changedPassword = await UserStories.ChangePasswordAsync(changePassword);
+            await UserStories.LoginExistingUserAsync(newClient, userName, $"!Test.Password2$");
+        }
```

Comment:
@HaoK Any suggestions on how to verify the user claim? Or how to check for a new cookie? (any other way to test using RefreshSignInAsync)

Reply:
I think the easiest thing to do is add a new page to https://github.com/aspnet/Identity/tree/dev/test/WebSites/Identity.DefaultUI.WebSite/Pages which just prints out the claim so you can confirm that its still there...